src/App.js

```diff
@@ -0,0 +1,22 @@
+import React, {Component} from 'react';
```

Comment:
Flow will check this file without `Flow` comment? If no, pls add comment here and in `index.js` to

Reply:
Done.